configuration/eclipseiotmarket.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: documentation
+title: Eclipse IoT Marketplace
+---
+
+{% include base.html %}
+
+# Eclipse IoT Marketplace Integration
+
+The <a href="https://marketplace.eclipse.org/taxonomy/term/4988%2C4396/title">Eclipse IoT Marketplace</a> hosts third-party add-ons that can be browsed and installed to openHAB through this integration.
```

Comment:
It works as html but the rest of the document is written with markdown, including the same link further below.